packages/service-provider-server/package.json

```diff
@@ -40,6 +40,7 @@
   "dependencies": {
     "@mongosh/errors": "^0.0.8",
     "@mongosh/service-provider-core": "^0.0.8",
+    "@mongosh/shell-api": "^0.0.8",
```

Comment:
Nice! I don't know ... it should work unless, as you mentioned, we are accidentally patching the wrong module (another `bson`) 🤔. I hope this approach is safe enough. I'm having few nightmares about the `Symbol` not being the same when it comes from inside the VM context or from a thread (if we want to offload the evaluation in the browser). If we see this would be shaky perhaps we'd have to consider something else, but nothing sounds so cheap.

Reply:
It's a bit awkward but not crazy that we would have to patch in two places - we patch once for the bson used by the driver, so that the documents returned by the driver are patched. Then we patch again for the bson library we load into the context that the user can create directly. It would be better if we could just do it once, but I understand why we would have to do it twice / there would be two modules.